src/commands/force/source/pull.ts

```diff
@@ -15,16 +15,17 @@ import {
   RetrieveResult,
   SourceComponent,
 } from '@salesforce/source-deploy-retrieve';
-import { ChangeResult, replaceRenamedCommands, SourceTracking } from '@salesforce/source-tracking';
-import { SourceCommand } from '../../../../sourceCommand';
-import { PullResponse, PullResultFormatter } from '../../../../formatters/source/pullFormatter';
-import { updateTracking, trackingSetup } from '../../../../trackingFunctions';
+import { ChangeResult, SourceTracking } from '@salesforce/source-tracking';
+import { SourceCommand } from '../../../sourceCommand';
+import { PullResponse, PullResultFormatter } from '../../../formatters/source/pullFormatter';
+import { updateTracking, trackingSetup } from '../../../trackingFunctions';
 
 Messages.importMessagesDirectory(__dirname);
 const messages: Messages = Messages.loadMessages('@salesforce/plugin-source', 'pull');
 
 export default class Pull extends SourceCommand {
   public static description = messages.getMessage('description');
+  public static aliases = ['force:source:beta:pull'];
```

Comment:
good idea, should we add a warning for deprecating the `beta` commands too?

Reply:
Is there a way for the command to know if it's being run from its alias?

Reply:
I just tried--it resolves it to the real command, not the alias.  😞